### DISCUSSION TITLE
Import existing Word/PDF file content and structure to DocFX?

### DISCUSSION THREAD

**Discussion by yooakim** [dotnet/docfx#9961] [Q&A]
Hi, I am new to DocFX. My use case is that we have a Microsoft Word document (or PDF) that is about 100 pages long.

It is a handbook and it has a good structure.

We would like to migrate this to DocFX.

I realise this is not so easy to do fully automated but I was wondering if people here have suggestions on how to do this?

I am very familiar with markdown but not so experienced in converting Word/PDF files. So any suggestions are welcome.

**Comment by filzrev** [comment]
One solution candidate would be to use [pandoc](https://github.com/jgm/pandoc) to extract `Markdown`, `Table of Content` and `Embedded medias` from Word documents.
Then manually edit/restructure content files.
